docker/activemq/activemq

```diff
@@ -0,0 +1,769 @@
+#!/bin/bash
+set -x
+
```

Comment:
why do we need this vendor file?  Did you make any changes?  I would think it would be part of the base image?

Reply:
Ah! I forgot to remove that... We don't need it, I just pulled it and added some tracing to do some debugging. I will remove it and test it again to make sure it works fine.
Thanks for spotting that!